src/main/java/uk/gov/hmcts/ccd/domain/service/search/global/GlobalSearchServiceImpl.java

```diff
@@ -117,13 +118,14 @@ public CrossCaseTypeSearchRequest assembleSearchQuery(GlobalSearchRequestPayload
     }
 
     public GlobalSearchResponsePayload transformResponse(final GlobalSearchRequestPayload requestPayload,
-                                                         final CaseSearchResult caseSearchResult) {
+                                                         CaseSearchResult caseSearchResult,
```

Comment:
If we're no longer using the cases from the `CaseSearchResult` then looks like we only need the total instead from it - could just pass `Long totalResults` here instead to avoid any potential confusion with there being two lists of `CaseDetails`.